### PR TITLE
libwin: Add utf8 compatible rmdir and mkdir wrappers.

### PR DESCRIPTION
**Pull request by Keith-Cancel**
Very simple wrappers that fixes one of the causes of mojibake.

If you see my other pull request you see that I noticed that the the windows versions of mkdir and rmdir are not compatible with utf8. So this pull should fix that issue. It was simple enough that I decided to fix it.

obvious

### CODE REVIEW

File: shared/windows/mkdir.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <windows.h>
+#include <wchar.h>
+
+typedef	unsigned short mode_t;
```
Comment (kuba160) [inline shared/windows/mkdir.c]: Please use space instead of tab

File: shared/windows/mkdir.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include <windows.h>
```
Comment (kuba160) [inline shared/windows/mkdir.c:24]: I know that there are no license headers but please add it to these files. Something like this is good:
```
/*
    shared/windows/mkdir.c
    Copyright (C) 2020 Keith Cancel

    This software is provided 'as-is', without any express or implied
    warranty.  In no event will the authors be held liable for any damages
    arising from the use of this software.

    Permission is granted to anyone to use this software for any purpose,
    including commercial applications, and to alter it and redistribute it
    freely, subject to the following restrictions:

    1. The origin of this software must not be misrepresented; you must not
     claim that you wrote the original software. If you use this software
     in a product, an acknowledgment in the product documentation would be
     appreciated but is not required.

    2. Altered source versions must be plainly marked as such, and must not be
     misrepresented as being the original software.

    3. This notice may not be removed or altered from any source distribution.
*/
```
I'm going to add other headers for the rest of the files.

File: shared/windows/rmdir.c [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (kuba160) [inline shared/windows/rmdir.c:1]: Same as previous one.

**Review by kuba160 — CHANGES_REQUESTED** [review]

File: shared/windows/mkdir.c (hunk 2 above)
Comment (Keith-Cancel) [reply shared/windows/mkdir.c:24]: Okay easy enough.

File: shared/windows/rmdir.c [hunk 4]
```diff
@@ -1,4 +1,27 @@
 
+/*
+    shared/windows/mkdir.c
```
Comment (kuba160) [inline shared/windows/rmdir.c]: `shared/windows/rmdir.c`

**Review by kuba160 — CHANGES_REQUESTED** [review]

File: shared/windows/mkdir.c (hunk 1 above)
Comment (Keith-Cancel) [reply shared/windows/mkdir.c]: Did not notice that guy. Easy fix though.

sneaky

File: shared/windows/rmdir.c (hunk 4 above)
Comment (kuba160) [reply shared/windows/rmdir.c]: mkdir.c should be changed to rmdir.c as it is different file

**Review by kuba160 — CHANGES_REQUESTED** [review]

**Comment by Keith-Cancel** [comment]
Did not even notice that lol. My bad.

**Pull request merged**